honeybee_radiance_command/options/rfluxmtx.py

```diff
@@ -36,3 +39,170 @@ def v(self):
     @v.setter
     def v(self, value):
         self._v.value = value
+
+
+class RfluxmtxControlParameters(object):
+    """Rfluxmtx ControlParameters.
+
+    Set the values for hemisphere sampling type, hemisphere up direction and output file
+    location. This class generates a string for rflumtx which should be included
+    with a receiver aperture group.
+
+    Here is a sample rfluxmtx control parmaters: ``#@rfluxmtx u=0,1,0 h=kf o=output.vmx``
+
+    Args:
+        sampling_type: Set hemisphere sampling type. Acceptable inputs for hemisphere
+            sampling type are:
+             * u for uniform.(Usually applicable for ground).
+             * kf for klems full.
+             * kh for klems half.
+             * kq for klems quarter.
+             * rN for Reinhart - Tregenza type skies. N stands for subdivisions and
+               defaults to 1.
+              * scN for shirley-chiu subdivisions.
+            Add a ``-`` in front of the input for left-handed coordinates. For more
+            information see rfluxmtx docs.
+            https://www.radiance-online.org/learning/documentation/manual-pages/pdfs/rfluxmtx.pdf/at_download/file
+        up_direction: Orient the "up" direction for the hemisphere using the indicated
+            axis or direction vector. Valid inputs are [-]{X|Y|Z|ux,uy,uz}. Default: Y
```

Comment:
You might want to clarify that this is a string since all of the other direction vectors in honeybee_radiance_command  are tuples of 3 floats.

Reply:
Actually, I see that it can be either a tuple or a string. So I recommend clarifying the axis is a string while the direction vector is a tuple of 3 floats.